scripts/run-draft-pr-scripts.ts

```diff
@@ -131,48 +150,18 @@ class DraftPRScriptRunner {
       }
       
       console.log(`✅ Found draft PR #${pr.number} for branch: ${this.branchName}`);
+      console.log(`✅ Found PR #${pr.number}, using current HEAD for script discovery`);
       return pr.number.toString();
     } catch (error) {
       throw new Error(`Failed to find PR for branch ${this.branchName}: ${error}`);
     }
   }
 
-  private async setupPREnvironment(): Promise<void> {
-    console.log(`🔍 Finding PR for branch: ${this.branchName}...`);
-    
-    this.prNumber = await this.findPRByBranch();
-    
-    console.log(`🔄 Checking out PR #${this.prNumber} branch...`);
-    
-    const githubToken = process.env.GITHUB_TOKEN;
-    const githubRepo = process.env.GITHUB_REPOSITORY;
-    
-    if (!githubToken || !githubRepo) {
-      console.log("⚠️ Not in GitHub Actions environment, skipping PR checkout");
-      return;
-    }
-    
-    try {
-      execSync(`git fetch origin pull/${this.prNumber}/head:pr-${this.prNumber}`, {
-        cwd: this.projectRoot,
-        stdio: 'inherit'
-      });
-      
-      execSync(`git checkout pr-${this.prNumber}`, {
-        cwd: this.projectRoot,
-        stdio: 'inherit'
-      });
-      
-      console.log(`✅ Successfully checked out PR #${this.prNumber} branch`);
-      
-    } catch (error) {
-      throw new Error(`Failed to setup PR environment: ${error}`);
-    }
-  }
+
 
   private getAddedFilesInPR(): string[] {
     try {
-      const output = execSync(`git diff --diff-filter=A --name-only origin/main...${this.branchName}`, {
+      const output = execSync(`git diff --diff-filter=A --name-only origin/main...HEAD`, {
```

Comment:
Wait you forgot to add the following two lines before this command:
- git status
- git log --oneline -5

Before this command so I can debug what the heck is going on here